### ISSUE TITLE
Error in handling inability to load Linux SSL module on Fedora Core 20

### ISSUE THREAD

**Issue by unwiredben** [wbond/package_control#863]
Starting Sublime Text 3.0.3067 with latest Package Control gives me this error on the console:

```
reloading plugin 0_package_control_loader.01-ssl-linux
Traceback (most recent call last):
  File "01-ssl-linux in /home/corp.roku/bcombee/.config/sublime-text-3/Installed Packages/0_package_control_loader.sublime-package", line 22, in <module>
ImportError: libssl.so.1.0.0: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/sublime_text_3/sublime_plugin.py", line 74, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/opt/sublime_text_3/sublime_plugin.py", line 678, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "01-ssl-linux in /home/corp.roku/bcombee/.config/sublime-text-3/Installed Packages/0_package_control_loader.sublime-package", line 26, in <module>
NameError: name 'e' is not defined
```

It looks like the two lines in 01-ssl-linux.py with

```
except (ImportError):
```

need to be

```
except (ImportError) as e:
```
instead, but I couldn't find the repo where that loader code was maintained.  With that change locally, I correctly get libssl.so.10 loaded.

**Comment by FichteFoll** [comment]
The exact repo is either [here](https://github.com/codexns/sublime-ssl-linux) or [here](https://github.com/codexns/linux-python-_ssl-build), where the latter only compiles the binaries and the former contains the errorneous `loader.py` code.

**Comment by wbond** [comment]
This should be fixed by PC 3.0.1, which was just released.
